tests/atomistics/structure/test_neighbors.py

```diff
@@ -339,6 +339,13 @@ def test_norm_order(self):
         with self.assertRaises(ValueError):
             neigh.norm_order = 3
 
+    def test_chemical_symbols(self):
+        basis = CrystalStructure("Fe", bravais_basis="bcc", lattice_constants=1)
```

Comment:
I see that `CrystalStructure` just invokes `_CrystalStructure` and that `_CrystalStructure` claims it exists only for historical reasons. I suggest but don't insist that we use `StructureFactory().bulk("Fe", a=1, cubic=True)` instead.